app/scripts/api.es6

```diff
@@ -1,10 +1,12 @@
 import ckan from './utils/ckan';
 
-
 const API_ROOT = '//data.takwimu.org/api';
 const resourceUrl = id => `${API_ROOT}/action/datastore_search?resource_id=${id}`;
 // const queryUrl = q => `${API_ROOT}/action/datastore_search_sql?sql=${q}`;
+const staticResource = q => `layers/${q}.json`;
 
+const get = url =>
+  fetch(url).then(resp => resp.json());
```

Comment:
:+1: we can maybe add move ['makeHTTPErrorNice`](https://github.com/WorldBank-Transport/waterdash/blob/waterpoints-loading/app/scripts/utils/ckan.es6#L18) and [`rejectIfNotHTTPOk`](https://github.com/WorldBank-Transport/waterdash/blob/waterpoints-loading/app/scripts/utils/ckan.es6#L30) somewhere to use it here also, after #8 is merged